src/App.js

```diff
@@ -1,26 +1,43 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import React from "react";
+import { Switch, Redirect, Route, withRouter } from "react-router-dom";
+import AuthenticationRoute from "./containers/AuthenticationRoute";
+import routes from "./helpers/routes";
+import LoginScreen from "./components/LoginScreen";
+import "./App.css";
+
+const App = props => {
+  const isAuthenticated = false;
 
-function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <Switch>
```

Comment:
There is a little problem which we discovered with Tigran while doing peer programming, the problem is related to internal Switch. In following example code will never reach /about route, because it will get stack in internal switch - in case of being authenticated user will be redirected to home and to login in case of not being authenticated.

```
<Switch>
      <AuthenticationRoute
        path="/login"
        withAuth={false}
        component={LoginScreen}
        redirectOnFailure="/home"
        isAuthenticated={isAuthenticated}
      />
      <AuthenticationRoute
        path="/"
        withAuth={true}
        redirectOnFailure="/login"
        isAuthenticated={isAuthenticated}
        render={() => (
          <Switch>
            {routes.map((route, index) => (
              <Route
                key={index}
                path={route.path}
                component={route.component}
              />
            ))}

            <Redirect to="/home" />
          </Switch>
        )}
      />
      <Route path="/about" component={() => <p>test</p>} />
    </Switch>
```